src/hotspot/share/utilities/resourceHash.hpp

```diff
@@ -95,9 +108,13 @@ class ResourceHashtable : public ResourceObj {
         }
         ++bucket;
       }
+      FREE_C_HEAP_ARRAY(Node*, _table);
     }
   }
 
+  unsigned size() const { return static_cast<const TABLE_IMPL*>(this)->size_impl(); }
```

Comment:
I think size() should return the number of entries. The number of buckets should use a different name (assuming it needs to be publically accessible).

Reply:
In the latest version, I am following the same naming convention in hashtable.hpp:
- table_size() = number of buckets
- number_of_entries() = number of entries

Reply:
Yes, please.